CryptoAnalysis/src/main/java/crypto/analysis/CrySLRulesetSelector.java

```diff
@@ -31,7 +33,7 @@ public String toString() {
 			rules.addAll(getRulesset(rulesBasePath, ruleFormat, s));
 		}
 		if (rules.isEmpty()) {
-			System.out.println("No CrySL rules found for rulesset " + set);
```

Comment:
Since you are replacing them in the whole PR i wonder what you think about a debug mode that could be enabled that prints these messages to stdout or stderr? We should log them of course in any case.

Btw. where are the logs written to? Do they get shown in the console? If yes, text above is obsolete.